src/attr.rs

```diff
@@ -29,7 +29,14 @@ pub fn display(attrs: &[Attribute]) -> Result<Option<Display>> {
                 _ => unimplemented!(),
             };
 
-            let lit = LitStr::new(lit.value().trim(), lit.span());
+            let lit = LitStr::new(
+                &lit.value()
+                    .trim()
+                    // Deal with rustc including unnecessary parts of the doc comment :(
+                    .replace("\n     *", "\n")
```

Comment:
this will probably need to handle various amounts of whitespace, since the doc comment could be on a type in an inline module for example. It's probably worth adding another test case with a higher level of indentation to ensure that we correctly strip the whitespace in both cases.

Reply:
Here comes the regexs... I'll add the test case and address it when I get a chance.

Reply:
So this can either be solved with a regex, or with code that has some knowledge about the formatting we're trying to work around. Something along the lines of if all lines in a comment except the first one start with the same amount of spaces followed by `*`, then strip that.
Do you have a preference?

Reply:
I was thinking split the string with `lines()`, then use `trim()` on each str, which should hopefully give `* <the rest of each comment line>` which can then be fixed with a consistent `replace("* ", "")` or something

Reply:
I initially thought about doing that but I wonder what happens in the following:
```
/// Some list:
/// * a
/// * b
/// * c
```
This could be detected by the comment not starting with a `*` (from the `/**` part).

Reply:
This might be okay since we're already banning these kinds of doc comments with your other change

Reply:
Fair point :)